docs/examples/UsingCalculationResults.rst

```diff
@@ -39,15 +39,15 @@ dimensions/coordinates:
 -  ``P``: pressures (in Pa).
 -  ``T``: temperatures (in K).
 -  ``component``: the string names of the components in the system
--  ``internal_dof``: The internal\_dof (internal degrees of freedom) is
-   the index of the site in any phase's site fraction array. Below the
-   FCC\_A1 phase has the sublattice model (AL, ZN) and thus the
-   internal\_dof are integers 0 and 1 referring to the AL site (index 0)
+-  ``internal_dof``: The internal_dof (internal degrees of freedom) is
+   the index of the site in any phase’s site fraction array. Below the
+   FCC_A1 phase has the sublattice model (AL, ZN) and thus the
+   internal_dof are integers 0 and 1 referring to the AL site (index 0)
```

Comment:
Don't variable names with underscores still need to be escaped?

Reply:
I built the docs locally and everything looks right
![Screen Shot 2020-02-06 at 8 29 20 PM](https://user-images.githubusercontent.com/7681751/73993166-67a67f00-491f-11ea-97b7-d8e3785d6866.png)